cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -145,7 +145,7 @@ void EditBoxImplCommon::setFontColor(const Color4B& color)
 
 void EditBoxImplCommon::setPlaceholderFont(const char* pFontName, int fontSize)
 {
-    this->setNativePlaceholderFont(pFontName, fontSize);
+    this->setNativePlaceholderFont(pFontName, fontSize * _labelPlaceHolder->getNodeToWorldTransform().a);
```

Comment:
`getNodeToWorldTransform` to world transform is a `Mat4` type,  we should change it to `getNodeToWorldAffineTransform`.